packages/react/src/hooks/utils.ts

```diff
@@ -29,6 +30,7 @@ export const createGetToken = (isomorphicClerk: IsomorphicClerk) => {
  * @internal
  */
 export const createSignOut = (isomorphicClerk: IsomorphicClerk) => {
+  // eslint-disable-next-line
```

Comment:
❓ Mind explicitly defining what we're disabling here? Thanks!

Reply:
same as above